docs/api/en/objects/InstancedMesh.html

```diff
@@ -179,7 +181,10 @@ <h3>
 		<p>
 			Sets the given local transformation matrix to the defined instance. Make
 			sure you set [page:.instanceMatrix][page:BufferAttribute.needsUpdate .needsUpdate] 
-			to true after updating all the matrices.
+			to true after updating all the matrices. The bounding volumes should be updated
+			whenever instances are transformed. Depending on how you transform instances, the
+			bounds can be larger or smaller than before and the bounding volumes should
+			reflect that.
```

Comment:
> The bounding volumes should be updated whenever instances are transformed.

This is not clear. What are the "bounding volumes"? And what do they have to do with the method being documented?

> Depending on how you transform instances, the bounds can be larger or smaller than before and the bounding volumes should reflect that.

That statement is not helpful, and it provides no useful information.

@Mugen87 is correct. It is best not to head down this slippery slope. The API docs are not intended to be a tutorial.

Reply:
It's understandable that devs who struggle with an issue for hours demand a more detailed documentation. 

Given that `BufferGeometry`, `InstanedMesh` and `SkinnedMesh` do not update their bounding volumes automatically, I think we should somewhere explain that it's the task of the app to recompute the bounding sphere and box when the underlying data (vertices, instance transformations or bones) change.

If we note add a note in the documentation pages for `BufferGeometry`, `InstanedMesh` and `SkinnedMesh`, where should we mention this detail? 

Considering how we documented other classes in the past (e.g. the restrictions of `ShaderMaterial`) I think the above pages are the right place. If you are not happy with the descriptions, maybe we can rephrase to make things more clear?

Reply:
>It's understandable that devs who struggle with an issue for hours demand a more detailed documentation.

Yes, that seems reasonable.

If you want to add a comment about the responsibility of "your app", I think it is appropriate to add it in the documentation the `computeBoundingBox/Sphere()` method _iteslf_.

You can link to "How to update things", instead -- or in addition --  if you want.

 I would revert this PR. I think you can do better, and I hope the OP would find it equally-acceptable.

Reply:
Okay, I have reverted the PR. I'll give it a new go tomorrow 👍 .